terraform/README.MD

```diff
@@ -0,0 +1,20 @@
+
+## Usage
+```
+export PROJECT_ID=<project_id>`
```

Comment:
What would be the command to execute only the BigQuery sub module? (to add it just for reference when multiple sub modules are present)

Reply:
Yes and No. It won't be able to use the highlevel config